src/Components/Decisions/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import './style.css';
+
+function Decisions() {
+  return (
+    <form>
+      <label className="decision-selection" htmlFor="status-decision" control="decisions">
+        Decisions
+        <select className="container" id="status-decision" name="decisions">
+          <option>Active</option>
+          <option>Inactive</option>
+          <option>Incomplete</option>
+          <option>Suspended</option>
+          <option>Closed</option>
+        </select>
+      </label>
+      <div className="decision-making">
+        <button className="buttonContainer" type="submit">Cancel</button>
```

Comment:
I don't think the cancel button should be of `type=submit`, if this component is the child of a form element, this will trigger a `submit()` I would just remove that attr 🙂 ✔️

Reply:
Updated type to "button"